src/ontology/mondo-ingest.Makefile

```diff
@@ -33,8 +33,8 @@ $(TMPDIR)/ordo_relevant_signature.txt: component-download-ordo.owl | $(TMPDIR)
 #######################################
 # This section is concerned with transforming the incoming sources into the
 # Monarch Ingest schema.
-# Illegal punning on some properties #60: https://github.com/monarch-initiative/omim/issues/60
-# todo: what does this have to do with #60 exactly? Does it address it? can that issue be closed?
+# todo: Illegal punning on some properties #60: https://github.com/monarch-initiative/omim/issues/60
+#  - what does this have to do with #60 exactly? Does it address it? can that issue be closed?
```

Comment:
No this issue need to be fixed at source, this is probably a statement that says: There is a workaround to this problem in the following lines